models/intermediate/account_balances/int_account_balances__offers.sql

```diff
@@ -57,30 +57,47 @@ with
             and (acc.valid_to is null or date(acc.valid_to) >= (select min(day) from dt))
     )
 
-select
-    dt.day
-    , tl.account_id
-    , tl.asset_type
-    , tl.asset_issuer
-    , tl.asset_code
-    , tl.balance
-from dt
-inner join filtered_tl as tl
-    on
-    dt.day >= date(tl.valid_from)
-    and (dt.day < date(tl.valid_to) or tl.valid_to is null)
 
-union all
+    , aggregate as (
+        select
+            dt.day
+            , tl.account_id
+            , tl.asset_type
+            , tl.asset_issuer
+            , tl.asset_code
+            , tl.balance
+        from dt
+        inner join filtered_tl as tl
+            on
+            dt.day >= date(tl.valid_from)
+            and (dt.day < date(tl.valid_to) or tl.valid_to is null)
+
+        union all
+
+        select
+            dt.day
+            , acc.account_id
+            , acc.asset_type
+            , if(acc.asset_type = 'native', 'XLM', acc.asset_issuer) as asset_issuer
+            , if(acc.asset_type = 'native', 'XLM', acc.asset_code) as asset_code
```

Comment:
technically you could hardcode `XLM` since it's the only balance stored in accounts